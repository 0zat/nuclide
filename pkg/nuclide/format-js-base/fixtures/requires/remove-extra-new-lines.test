/**
 * Header comment.
 */

var a = require('a');

var b = require('b');

var c = require('c');

var D = require('D');

var E = require('E');

var F = require('F');


a(b, c, D, E, F);
