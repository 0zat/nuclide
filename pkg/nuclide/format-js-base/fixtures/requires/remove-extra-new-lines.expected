/**
 * Header comment.
 */

var D = require('D');
var E = require('E');
var F = require('F');

var a = require('a');
var b = require('b');
var c = require('c');

a(b, c, D, E, F);
